src/screens/ProfileScreen/DetailScreen/ActiveTaskDetail.tsx

```diff
@@ -119,6 +89,9 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     borderRadius: 15,
   },
+  backButton: {
+    color: 'white',
```

Comment:
use color variable

Reply:
for color variable, we need to add a new issue  as we have to update it everywhere with testing.